pkg/rancher-desktop/components/Images.vue

```diff
@@ -79,13 +79,14 @@
 </template>
 
 <script>
+
+import SortableTable from '@pkg/components/SortableTable';
```

Comment:
Yeah, this is quite odd.. 

I had originally thought that this might have something to do with Vue files that aren't using typescript or imports that are an implicit `index` but neither of those hunches appears to check out. I'll see if I can't understand why these are impacted while others aren't.